frontend-site/src/views/Services.vue

```diff
@@ -8,8 +8,7 @@
             <v-layout row>
```

Comment:
```suggestion
            <v-layout row align-center>
              <v-flex xs5>
                <v-img :src="service.image"/>
              </v-flex>
```

Adding the `align-center` attribute to `v-layout` and moving the `v-img` out of the `div` should fix your problems with the image staying at the top of the card.

![image](https://user-images.githubusercontent.com/16447522/49322148-69caad80-f4d2-11e8-9883-cdaf8ff91cfa.png)

![image](https://user-images.githubusercontent.com/16447522/49322152-7818c980-f4d2-11e8-9d75-df0b9194069c.png)